pkg/destroy/openstack/openstack.go

```diff
@@ -422,6 +452,28 @@ func deletePorts(opts *clientconfig.ClientOpts, listOpts ports.ListOpts, logger
 				}
 			}
 
+			// If there is a security group created by cloud-provider-openstack we should find it and delete it.
+			// We'll look through the ones on each of the ports and attempt to remove it from the port and delete it.
+			// Most of the time it's a conflict, but last port should be guaranteed to allow deletion.
+			// TODO(dulek): Currently this is the only way to do it and if delete fails there's no way to get back to
+			//              that SG. This is bad and we should make groups created by CPO tagged by cluster ID ASAP.
```

Comment:
Is this issue being tracked somewhere so we don't forget about it?

Reply:
https://issues.redhat.com/browse/OSASINFRA-3200. This is a broader problem really, we don't have a good way to distinguish LBs between clusters too and currently we can have clashes by names in CPO. :/